dvc/remote/s3.py

```diff
@@ -81,22 +74,10 @@ def prefix(self):
 
     @property
     def s3(self):
-        if not self.creds:
-            session = boto3.session.Session(profile_name=self.profile)
-        else:
-            session = boto3.session.Session(
-                             aws_access_key_id=self.aws_access_key_id,
-                             aws_secret_access_key=self.aws_secret_access_key,
-                             region_name=self.region)
-        return session.client('s3', endpoint_url=self.endpoint_url)
+        session = boto3.session.Session(profile_name=self.profile,
```

Comment:
Similar question - what should take precedence in this case - env var (AWS_PROFILE) or our config file? We should document this stuff at least.